x-pack/plugins/enterprise_search/server/lib/pipelines/create_pipeline_definitions.test.ts

```diff
@@ -82,8 +83,7 @@ describe('formatMlPipelineBody util function', () => {
             field: '_source._ingest.processors',
             value: [
               {
-                model_id: modelId,
-                model_version: modelVersion,
```

Comment:
I think we want to keep this. From [Slack](https://elastic.slack.com/archives/C02U50QNEAG/p1663958344652809):
> We’ll keep `model_version` as it’s not a security concern